.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "submodules/RtMidi/src"]
 	path = submodules/RtMidi/src
 	url = https://github.com/thestk/rtmidi
+[submodule "BambooTracker/chip/emu2149"]
+	path = BambooTracker/chip/emu2149
+	url = https://github.com/rerrahkr/emu2149.git
+	branch = opna-ssg
```

Comment:
Can this be moved to `/submodules/emu2149` instead so all submodules are consistently in one place?

This will also require a small change to the readmes[1] (and maybe a mention in the update notes): Currently you can clone the repository / download an archive of it from the GitHub UI without any submodules and `.git` data if you build against the system's Rt libraries - which brings with it an improvement in download size and purity. This new module will now require the fetching of submodules / manual cloning of [rerrahkr/emu2149](https://github.com/rerrahkr/emu2149).

[1]: From the English readme:
> ```sh
> # Can leave away --recurse-submodules if CONFIG+=system_rtaudio AND CONFIG+=system_rtmidi will be used
> git clone https://github.com/BambooTracker/BambooTracker --recurse-submodules
> ```